templates_cspm_eventbridge/FullInstall.yaml

```diff
@@ -0,0 +1,100 @@
+AWSTemplateFormatVersion: "2010-09-09"
+Description: Creates IAM role for CSPM and EventbridgeRole, EventBridgeRule
+
+Metadata:
+  AWS::CloudFormation::Interface:
+    ParameterGroups:
+      - Label:
+          default: ""
+        Parameters:
+          - RoleName
+          - EventBridgeRoleName
+      - Label:
+          default: "Sysdig Settings (Do not change)"
+        Parameters:
+          - ExternalID
+          - TrustedIdentity
+          - EventBusARN
+
+    ParameterLabels:
+      RoleName:
+        default: "Role Name"    
+      ExternalID:
+        default: "External ID (Sysdig use only)"
+      TrustedIdentity:
+        default: "Trusted Identity (Sysdig use only)"
+      EventBusARN:
+        default: "Target Event Bus (Sysdig use only)"
+      EventBridgeRoleName:
+        default: "Unique name for Role and Policy created"       
+
+Parameters:
+  RoleName:
+    Type: String
+    Default: "sysdig-secure"
+    Description: The read-only IAM Role that Sysdig will create
+  EventBridgeRoleName:
+    Type: String
+    Default: "sysdig"
+    Description: Unique name for role and policy
```

Comment:
I'm not sure this needs to be configurable or if it should be hardcoded. @cgeers the EB policy/rules don't key off the EB Rule/Role name right?

Reply:
Can we use `sysdig-threat-detection` in the resource creation and remove this as a parameter?

Reply:
default values is provided and removed it from parameters list. Pls check the screenshot for full-install @nkraemer-sysdig